operator/eni.go

```diff
@@ -142,7 +142,7 @@ func startENIAllocator(awsClientQPSLimit float64, awsClientBurst int, eniTags ma
 	}
 
 	// Initial blocking synchronization of all ENIs and subnets
-	instances.Resync()
+	instances.Resync(ctx)
```

Comment:
If this is going to use the context and silently return whether it was successful or not, then we should really check the context at some point here to make sure that we signal that actually, the ENI allocator is *not* started, and the reason it isn't was the connectivity problem with the AWS metadata service.

(Does it make sense to actually put a real context through this startup piece anyway? That could force the pod into `CrashLoopBackoff` during startup rather than sitting and waiting until proper access to the AWS API is established)

Reply:
This already did silently return whether it was successful or not - this simply provides means to plumb down 'stopping' of API calls if the context provided to the function is canceled. The only caller of this function uses `context.TODO()` right now. If you like, I can simply remove the context parameter here and put `context.TODO()` in the call to `instances.Resync` here, so that it's known that we are not actually providing any useful context yet.

Reply:
This would address my concern.

Reply:
Done, `startENIAllocator` no longer takes in a `context.Context`, and the referenced call to `instances.Resync` has been changed to take `context.TODO()`.